src/Swan.Lite/Extensions.Enumerable.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Swan
+{
+    /// <summary>
+    /// This class contains extensions methods for types implementing IEnumerable<T>
+    /// </summary>
+    public static class EnumerableExtensions
+    {
+        /// <summary>
+        /// This method returns the <see cref="System.Linq.Enumerable.Union"/>
+        /// of all non-null parameters.
+        /// </summary>
+        /// <returns> An <see cref="IEnumerable{T}"/> that contains the elements from non-null input sequences, excluding duplicates. </returns>
+        public static IEnumerable<T> UnionExcludingNulls<T>(this IEnumerable<T> @this, IEnumerable<T> second)
+        {
+            if (@this == null) return second ?? Enumerable.Empty<T>();
+            return second == null ? @this : @this.Union(second);
+        }
```

Comment:
If both `this` and `second` are null, you are returning `null` here.

This may be exactly what you wanted, but then why write this method?

If instead you want to return an empty enumerable when both parameters are null, plus not have to worry about checking each parameter, you can do this:
```C#
        public static IEnumerable<T> UnionExcludingNulls<T>(this IEnumerable<T> @this, IEnumerable<T> second)
            => Enumerable.Union(
                @this ?? Enumerable.Empty<T>(),
                second ?? Enumerable.Empty<T>());
```

It may take a handful of nanoseconds longer, but it's way cleaner, and the intent is clear from the code.